util/genrest/resttools/populatefield.go

```diff
@@ -214,3 +241,22 @@ func findProtoFieldByJSONName(fieldName string, fields protoreflect.FieldDescrip
 	}
 	return result, nil
 }
+
+func parseWellKnownType(message protoreflect.Message, fieldDescriptor protoreflect.FieldDescriptor, value string) (*protoreflect.Value, error) {
+	messageFieldTypes := message.Type().(protoreflect.MessageFieldTypes)
```

Comment:
For safety/future-proofing, I would add a second return param `ok` here to prevent a panic, and return an error if `!ok`

Reply:
Done.